plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -93,7 +93,13 @@ data class BazelConfig(
      * Use Conan2 when fetching Conan packages, otherwise use Conan 1.
      */
     @OrtPluginOption(defaultValue = "false")
-    val useConan2: Boolean
+    val useConan2: Boolean,
+
+    /**
+     * Only scan Bazel dependencies and skip reporting Conan packages in the dependency tree.
+     */
+    @OrtPluginOption(defaultValue = "false")
+    val bazelDependenciesOnly: Boolean
```

Comment:
Related to my comment on the previous commit, can't we archive the same by simply disabling Conan as a package manager? Or do you think there are use-cases where there *are* both Bazel and Conan projects in one repository, but they should be recognized independently?

Reply:
This is a case with one of our customer: they have a repository with Bazel and Conan definition files + Conan packages in the Bazel project.

Additionally I created the `bazelDependenciesOnly` parameter because it also exist in `Pub`, so you can see it as some kind of alignment.